### PR TITLE
Separate clockConfigRead and clockConfigWrite, Add GPIO_TCLKA_TCLKC_SEL

### PR DESCRIPTION
**Pull request by bahadirdonmez**

### CODE REVIEW

File: port/board/afck/cli_commands.c [hunk 1]
```diff
@@ -1,700 +1,854 @@
...
+
+    // Print the current enable mask
+    printf("\r\nCurrent enable mask: 0x%04X\r\n", enableMaskValue);
+
+    // Print the current clock output map
+    printf("Current clock output map: 0x");
+    for (uint8_t i = numberOfClocks - 1; i < numberOfClocks; i--) {
+        // Extract each 4-bit segment and print as a hexadecimal digit
+        uint8_t segment = (outputMapValue >> (4 * i)) & 0xF;
+        printf("%X", segment);
```
Comment (mastadle) [inline port/board/afck/cli_commands.c:649]: Can't you just print `outputMapValue` as `%016X` ?

**Comment by mastadle** [comment]
Don't you want to include `GPIO_TCLKA_TCLKC_SEL` in the gpio set/read command in the cli commands?

File: port/board/afck/cli_commands.c (hunk 1 above)
Comment (bahadirdonmez) [reply port/board/afck/cli_commands.c:649]: Yeah this was a bit annoying, I tried the followings but none of them worked...

```
printf("Current clock output map: 0x%016X\r\n", outputMapValue);
printf("Current clock output map: 0x%016lX\r\n", outputMapValue);
printf("Current clock output map: 0x%016llX\r\n", outputMapValue);
printf("Current clock output map: 0x%016" PRIx64 "\r\n", outputMapValue);
```

Could it be that the micro-controller cannot print long long unsigned integer (64-bit)?

**Comment by bahadirdonmez** [comment]
> Don't you want to include `GPIO_TCLKA_TCLKC_SEL` in the gpio set/read command in the cli commands?

Good point, I will add it quickly

**Comment by bahadirdonmez** [comment]
> Don't you want to include `GPIO_TCLKA_TCLKC_SEL` in the gpio set/read command in the cli commands?

I added and tested it quickly. It works.

**Comment by bahadirdonmez** [comment]
I now added gpio_direction_read and gpio_direction_write CLI functions. They are necessary to dynamically set the GPIO_TCLKA_TCLKC_SEL pin as input in order to disable the low-jitter clock bypass.

File: port/board/afck/cli_commands.c (hunk 1 above)
Comment (mastadle) [reply port/board/afck/cli_commands.c:649]: You would have to add `printlli`, similar to https://github.com/tiqi-group/openMMC/blob/feature/afck/modules/printf-stdarg.c#L127 (and adapt `print` accordingly)

But there is no real reason to use a custom printf implementation. 
https://github.com/eyalroz/printf/ would be a better alternative.

At least write a comment why you did this.

Comment (bahadirdonmez) [reply port/board/afck/cli_commands.c:649]: Alright, I now added some comments about this and added a TODO for using https://github.com/eyalroz/printf/

**Pull request merged**